app/static/js/volunteerProfile.js

```diff
@@ -86,17 +86,27 @@ $(document).ready(function(){
     });
   });
 
-
-  $(".backgroundCheck").change(function () { // Updates the Background check of a volunteer in the database
-    checkType = $(this).attr("id")
+  $(".savebtn").click(function () { // Updates the Background check of a volunteer in the database
+    let bgCheckType = $(this).data("id")
+    let bgDate = $("#" + bgCheckType + "_date").val()
+    let checkPassed = $("[data-id=" + bgCheckType + "]").val()
+
+    if (checkPassed == 0 && bgDate != '' ) {
+        displayMessage("Y/N<br>Empty!", "danger")
+        return
+    } else if(bgDate == '' && checkPassed != 0) {
+        displayMessage("Date<br>Empty!", "danger")
+        return
+    }
+    else {
+        displayMessage("Saved!", "success")
```

Comment:
This one is premature. We don't want to send a success message before we've even tried the ajax request. We have `success` and `error` functions on the ajax request object for this.

Reply:
resolved